src/Robo/Plugin/Commands/LintPhpCommand.php

```diff
@@ -25,7 +25,7 @@ public function phpcsRun(ConsoleIO $io) {
     $installComposerPackages = FALSE;
     $createPhpCsFile = FALSE;
     if (!isset($composerJson['require-dev']['drupal/coder'])) {
-      $installComposerPackages = $io->confirm("You don't have the Coder composer package installed, Do you want us to install them for you (Required for linting)?", TRUE);
+      $installComposerPackages = $io->confirm("You don't have the Coder composer package installed, Do you want us to install it for you (Required for linting)?", TRUE);
     }
     if (!file_exists($this->getLocalEnvRoot() . '/phpcs.xml.dist')) {
```

Comment:
I think we want `phpcs.xml`, not `.dist`.  Same goes for the lists of tasks below.